AK/UUID.h

```diff
@@ -32,7 +32,7 @@ class UUID {
     ~UUID() = default;
 
     bool operator==(const UUID&) const;
-    bool operator!=(const UUID& other) const { return !(*this == other); }
+    bool operator!=(const UUID& other) const = default;
```

Comment:
do we need = default here b/c the operator is not inline in the header? side question, why is operator== not inline here? and can it be defaulted? or can we delete this.

Reply:
It's not needed. I added it to ensure that no one will look at this code, notice that `operator!=` is "missing", and re-add it. Alternatively, I can remove the rest of the other deleted operators.

I checked and it does compile with `operator==` being defaulted and everything else removed. It's up to you; if you think that would be better, I can change the PR.

Reply:
^ did that